testsuite/gnat2goto/tests/discriminated_record/discriminated_record.adb

```diff
@@ -15,4 +15,6 @@ procedure Discriminated_Record is
 begin
    Inst1.A (5) := 1;
    Inst1.B (10) := 2;
+
+   pragma Assert (Inst1.A (5) = 1);
```

Comment:
This I very much approve of.  If you update `B` and then check the same assertion again it would be even better!